src/ai_code_review/git_cli.py

```
@@ -0,0 +1,71 @@
+        base (str): The base commit hash or branch.
+        head (str): The head commit hash or branch.
+        file_path (str): The path to the file in the repository.
+
+    Returns:
+        str: The diff patch for the specified file.
+
+    """
+    return _run_git_cmd(repo, "diff", "-p", "-U3", "-M", "-C", f"{base}...{head}", "--", file_path)
+
+
+def show_text(repo: str, rev: str, path: str) -> tuple[str, str | None]:
```

Ruff (0.12.2)

Missing docstring in public function

(D103)